python/revng/api/manager.py

```diff
@@ -303,8 +303,6 @@ def _run_analysis(
             options._string_map,
         )
 
-        if result != ffi.NULL and not _api.rp_diff_map_is_empty(result):
```

Comment:
In the commit message state why we do this and what needs to be done to re-enable this behavior.

Reply:
Done